tests/pools/test_pool_rpc.py

```diff
@@ -693,10 +691,14 @@ async def test_absorb_pooling(self, one_wallet_node_and_rpc, fee, trusted, bt, s
             assert (250000000000 + fee) in [tx.additions[0].amount for tx in tx1]
 
     @pytest.mark.asyncio
-    @pytest.mark.parametrize("trusted", [True])
-    @pytest.mark.parametrize("fee", [0])
-    async def test_self_pooling_to_pooling(self, setup, fee, trusted, self_hostname):
-        """This tests self-pooling -> pooling"""
+    @pytest.mark.parametrize("trusted_and_fee", [(True, 0), (False, 0)])
```

Comment:
`fee` is always 0 here...

Reply:
I see. in fact, in the previous version we only test this with `trusted=True` and `fee=0`. You added one more case.
But `fee` should probably not be a parameter at all to this test then